### ISSUE TITLE
Embedding-Guide removed

### ISSUE THREAD

**Issue by asalom** [route-me/route-me#184]
Hi,

The embedding guide hosted at the Wiki was removed a few months ago.

Here is the link: https://github.com/route-me/route-me/wiki/Embedding-Guide

**Comment by harry-wood** [comment]
Yeah what's happened to that?  Was it ever copied to github?  I see it's still here on the old google site: http://code.google.com/p/route-me/wiki/EmbeddingGuide
